styleguide/source/assets/js/tabs.js

```diff
@@ -90,14 +89,23 @@
        * @param {jQuery Object} $tabObj The element which has the .tab() function attached.
        * @param {string} linkHash
        */
-      function switchTabs($tabObj, linkHash) {
+      function switchTabs($tabObj, link) {
+
+        var linkHash = link.getAttribute("href");
         var widget = $tabObj.data('ui-tabs');
 
         var tabHash, positionInTab;
         var parts = linkHash.split('-');
         tabHash = parts[0];
         if (parts.length > 1) {
           positionInTab = parts[1];
+        } else {
+          // If old link, try to determine position from link text
+          var text = link.innerText;
+          var matches = link.innerText.match(/([0-9]+)/g);
```

Comment:
var matches = text.innerText.match(/([0-9]+)/g);

Reply:
I just removed the `text` variable since it was unused.